modules/git/foreachref/parser_test.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package foreachref_test
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+
+	"code.gitea.io/gitea/modules/git/foreachref"
+	"code.gitea.io/gitea/modules/json"
+	"github.com/stretchr/testify/require"
```

Comment:
Please place a blank line between internal packages and external packages.

Reply:
fixed here: 13cc3fb72 f67059144